x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -266,6 +267,30 @@ class PackagePolicyService {
     };
   }
 
+  public async listIds(
+    soClient: SavedObjectsClientContract,
+    options: ListWithKuery
+  ): Promise<ListResult<string>> {
+    const { page = 1, perPage = 20, sortField = 'updated_at', sortOrder = 'desc', kuery } = options;
+
+    const packagePolicies = await soClient.find<{}>({
```

Comment:
pass type 
```suggestion
    const packagePolicies = await soClient.find<PackagePolicySOAttributes>({
```

Reply:
oh, nevermind, I see that we are not expecting any fields to be returned in the response here due to `fields: []`

Reply:
It would be incorrect type, I specifically specify ```fields: []``` to avoid loading any attributes as I only need ids.